src/game/client/components/menus_settings.cpp

```diff
@@ -1664,12 +1664,24 @@ void CMenus::RenderSettingsSound(CUIRect MainView)
 					g_Config.m_SndRate = 48000;
 			}
 
-			m_NeedRestartSound = !s_SndEnable || s_SndRate != g_Config.m_SndRate;
+			m_NeedRestartSound = s_SndRate != g_Config.m_SndRate;
```

Comment:
turn on sndinit, change sndrate to sth else and then sndrate back - should still show the restart needed right?

Reply:
Damn, I couldn't figure why `!s_SndEnable` was needed there. That's what I was missing.

Reply:
Ah, you're right